README.md

```diff
@@ -1,72 +1,74 @@
+# SourceCoop
+
 [![CI](https://github.com/ampreeT/SourceCoop/actions/workflows/plugin.yml/badge.svg)](https://github.com/ampreeT/SourceCoop/actions/workflows/plugin.yml)
 [![GitHub Release](https://img.shields.io/github/v/release/ampreeT/SourceCoop?style=flat&label=Release&labelColor=%232C3137&color=%23EB551B)](https://github.com/ampreeT/SourceCoop/releases/latest)
 [![Discord](https://img.shields.io/discord/973591793117564988.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/Fh77rxQaEB)
 
-# SourceCoop
+| [Installation Guide](#installation-guide) - [Campaign Support](#campaign-support) - [Contributing](#contributing) - [Credits](#credits)
+:--: |
+| [Features & Configuration](https://github.com/ampreeT/SourceCoop/wiki/Features-&-Configuration) - [Server Running Tips](https://github.com/ampreeT/SourceCoop/wiki/Server-running-tips) - [Public Servers](https://github.com/ampreeT/SourceCoop/wiki/Public-Servers) |
+|  [Developing](https://github.com/ampreeT/SourceCoop/wiki/Developing) - [EDT Map Script Format](https://github.com/ampreeT/SourceCoop/wiki/EDT---Map-script-format) - [Authoring Maps](https://github.com/ampreeT/SourceCoop/wiki/Authoring-maps-for-SourceCoop) |
 
-SourceCoop is a cooperative mod for multi-player Source Engine games, that
-- Enables players to play together on single-player campaigns or maps
-- Complements existing cooperative campaigns or maps
-- Fixes various crashes
-- Re-enables single-player functionality
-- Includes map configs adjusting for cooperative play
-- Comes with optional addons, enhancing your experience
-- Can be used by server owners as the ultimate map config tool
-- Is built on [SourceMod](https://www.sourcemod.net)
-- __Only needs to be installed on the server__.
+SourceCoop is a cooperative mod for Source Engine games enabling singleplayer campaigns to be played together. It currently supports [Black Mesa](https://store.steampowered.com/app/362890/Black_Mesa/) and [Half-Life 2: Deathmatch](https://store.steampowered.com/app/362890/Black_Mesa/).
```

Comment:
and [Half-Life 2: Deathmatch](https://store.steampowered.com/app/362890/Black_Mesa/).

Reply:
wrong link

Reply:
Fixed the link for Half-life 2: Deathmatch.